src/WebJobs.Script/Description/Node/NodeFunctionInvoker.cs

```diff
@@ -372,9 +347,9 @@ protected override void OnScriptFileChanged(object sender, FileSystemEventArgs e
                 FunctionBinding.ConvertStreamToValue((Stream)input, dataType, ref input);
             }
 
-            Utility.ApplyBindingData(input, binder.BindingData);
-            var bindingData = NormalizeBindingData(binder.BindingData);
-            bindingData["invocationId"] = functionExecutionContext.InvocationId.ToString();
+            Utility.ApplyBindingData(input, invocationContext.Binder.BindingData);
```

Comment:
Similarly to TraceWriter -- C# does not add a Binder parameter, so it won't always be there.

---
In reply to: [77688076](https://github.com/Azure/azure-webjobs-sdk-script/pull/604#discussion_r77688076) [](ancestors = 77688076)